src/yuzu/main.cpp

```diff
@@ -1797,6 +1801,28 @@ void GMainWindow::OnMenuReportCompatibility() {
     }
 }
 
+void GMainWindow::OpenURL(QString const& url_str) {
```

Comment:
```suggestion
void GMainWindow::OpenURL(const QUrl& url) {
```

can be used to avoid needing the temporary